www_src/components/modal/modal.less

```diff
@@ -0,0 +1,70 @@
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0,0,0,0.3);
```

Comment:
That becomes `rgba(26, 34, 55, 0.007)`, which is barely visible. Did you have a typo there?